.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Compile parsers Windows
         if: matrix.os == 'windows-2022'
         run: |
-          Neovim\\bin\\nvim.exe --headless -c "TSInstallSync all" -c "q"
+          Neovim\\bin\\nvim.exe --headless -c "lua require'nvim-treesitter.install'.prefer_git=false" -c "TSInstallSync all" -c "q"
```

Comment:
seems this is unwanted change ?

Reply:
thanks for noting! but it was actually necessary to make CI pass. We can try with git but at the moment it's a bit luck (also with curl)